src/functional/ffront/decorator.py

```diff
@@ -119,6 +123,31 @@ def from_function(
             definition=definition,
         )
 
+    def _lowered_funcs_from_closureref(
+        self, closure_refs: ClosureRefs
+    ) -> list[itir.FunctionDefinition]:
+        lowered_funcs = []
+
+        vars_ = collections.ChainMap(closure_refs.globals, closure_refs.nonlocals)
+        for name, val in vars_.items():
```

Comment:
Just a suggestion, I think using `refs` instead of `vars_` and `value` instead of `val` would make it easier to read.

Reply:
I have left `vars_` unchanged which is in line with the rename in https://github.com/GridTools/gt4py/pull/705, but changed `val` to `value`.